premerge assertion failure -  `test_models.py::TestModelConfig::test_get_serve_options_without_accelerator_type`

```

[2025-08-21T22:20:55Z] E       AssertionError: assert [{'CPU': 1, 'GPU': 1}] == [{'CPU': 1, '...}, {'GPU': 1}]
--
  | [2025-08-21T22:20:55Z] E         At index 0 diff: {'GPU': 1, 'CPU': 1} != {'CPU': 1, 'GPU': 0}
  | [2025-08-21T22:20:55Z] E         Right contains one more item: {'GPU': 1}
  | [2025-08-21T22:20:55Z] E         Full diff:
  | [2025-08-21T22:20:55Z] E         - [{'CPU': 1, 'GPU': 0}, {'GPU': 1}]
  | [2025-08-21T22:20:55Z] E         ?                    ------------
  | [2025-08-21T22:20:55Z] E         + [{'CPU': 1, 'GPU': 1}]
  | [2025-08-21T22:20:55Z]
  | [2025-08-21T22:20:55Z] python/ray/llm/tests/serve/cpu/configs/test_models.py:216: AssertionError
  | [2025-08-21T22:20:55Z] =========================== short test summary info ============================
  | [2025-08-21T22:20:55Z] FAILED python/ray/llm/tests/serve/cpu/configs/test_models.py::TestModelConfig::test_get_serve_options_without_accelerator_type


```

--- a/python/ray/llm/tests/serve/cpu/configs/test_models.py
+++ b/python/ray/llm/tests/serve/cpu/configs/test_models.py
@@ -213,10 +213,7 @@ class TestModelConfig:
             "initial_replicas": 1,
             "max_replicas": 10,
         }
-        assert serve_options["placement_group_bundles"] == [
-            {"CPU": 1, "GPU": 0},
-            {"GPU": 1},
-        ]
+        assert serve_options["placement_group_bundles"] == [{"CPU": 1, "GPU": 1}]
         assert serve_options["placement_group_strategy"] == "STRICT_PACK"
         assert serve_options["name"] == "Test:test_model"
 

